scenarios/conftest.py

```diff
@@ -0,0 +1,28 @@
+import time
+import pytest
+
+# Dictionary to store method execution times
+method_times = {}
+
+@pytest.hookimpl(tryfirst=True)
+def pytest_runtest_makereport(item, call):
+    if call.when == 'call':
+        method_name = item.name
+        if method_name not in method_times:
+            method_times[method_name] = []
+        # Define the number of repetitions
+        repetitions = 3
+        for _ in range(repetitions):
+            start_time = time.time()
+            item.ihook.pytest_pyfunc_call(pyfuncitem=item)
```

Comment:
This is where we can run the function under inspection. Currently, this runs for all tests, so we might not want that.